.github/workflows/scala3-build.yml

```diff
@@ -8,9 +8,33 @@ on:
     tags-ignore: [ v.* ]
 
 jobs:
-  compile-and-test-with-scala3:
-    name: Compile and test with Scala 3
+  test:
+    name: Test
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        command:
```

Comment:
We can have a dynamic matrix here:

https://github.blog/changelog/2020-04-15-github-actions-new-workflow-features/#new-fromjson-method-in-expressions

I mean, have an sbt task that generates a JSON listing the Scala 3 projects, and generates the matrix dynamically using `fromJson`.